### ISSUE TITLE
Template sanity check for Send a Webpage

### ISSUE THREAD

**Issue by justingit** [justingit/dada-mail#1103]
If there's any template syntax errors in messages grabbed from a URL, they need to be shown before you try to send them out as a mass message. If you don't, a mass mailing will fail in mysterious ways to the end use

**Comment by justingit** [comment]
This problem can be recreated without using send url, but just pasting the bad link in the regular HTML text box.

**Comment by justingit** [comment]
OK, the problem isn't that the unescape template tags isn't done before template validation, it's that there is an issue with unescaping the template tags. If the lines don't wrap correctly, the template tags will be unescaped incorrectly. 

This may just mean the regex needs to match multiple lines.

**Comment by justingit** [comment]
This str will cause problems, but will get past the template validator, 

```
This is all that's needed: 


<a href="&lt;!-- tmpl_var list_unsubscribe_link __&gt;">blah</a>
```

**Comment by justingit** [comment]
This has been fixed in v11.18.1
